ui/push-health/Navigation.jsx

```diff
@@ -3,30 +3,34 @@ import PropTypes from 'prop-types';
 import { Navbar } from 'reactstrap';
 
 import LogoMenu from '../shared/LogoMenu';
+import { getJobsUrl } from '../helpers/url';
 import Login from '../shared/auth/Login';
-import SimpleTooltip from '../shared/SimpleTooltip';
+
+import { resultColorMap } from './helpers';
 
 export default class Navigation extends React.PureComponent {
   render() {
-    const { user, setUser, notify } = this.props;
+    const { user, setUser, result, notify, repo, revision } = this.props;
+    const overallResult = result ? resultColorMap[result] : 'none';
 
     return (
       <Navbar dark color="dark">
         <LogoMenu menuText="Push Health" />
-        <SimpleTooltip
-          text="[---PROTOTYPE---]"
-          textClass="text-white"
-          placement="bottom"
-          tooltipText={
-            <div>
-              This prototype is still in-progress. The section on `Tests` has
-              been implemented, but Linting, Coverage, Builds, etc are not. You
-              will notice in Treeherder that the status may say `OK` for a push
-              that has failed Builds or linting. These features will be updated
-              in the weeks to come.
-            </div>
-          }
-        />
+        <h4>
```

Comment:
This link seemed better in the nav bar.  It looked less busy this way, somehow.  We also don't need the "Prototype" info anymore.